documentation/ipi-install/modules/ipi-install-config-yaml.adoc

```diff
@@ -27,38 +27,43 @@ platform:
           username: +<user>+
           password: +<password>+
         xref:bootMACAddress[bootMACAddress]: +<NIC1-mac-address>+
-        xref:hardwareProfile[hardwareProfile]: default
+        rootDeviceHints:
+         deviceName: "/dev/sda"
       - xref:name[name]: openshift-master-1
         xref:role[role]: master
         xref:bmc-addressing_{context}[bmc]:
           address: ipmi://+<out-of-band-ip>+
           username: +<user>+
           password: +<password>+
         xref:bootMACAddress[bootMACAddress]: +<NIC1-mac-address>+
-        xref:hardwareProfile[hardwareProfile]: default
+        rootDeviceHints:
+         deviceName: "/dev/sda"
       - xref:name[name]: openshift-master-2
         xref:role[role]: master
         xref:bmc-addressing_{context}[bmc]:
           address: ipmi://<out-of-band-ip>
           username: +<user>+
           password: +<password>+
         xref:bootMACAddress[bootMACAddress]: +<NIC1-mac-address>+
-        xref:hardwareProfile[hardwareProfile]: default
+        rootDeviceHints:
```

Comment:
I think that this should include an xref to a relevant entry in the table  at ipi-install-additional-install-config-parameters.adoc explaining the rootDeviceHints and linked with each  ocurrence.

Shouldn't the /dev/sda be something like +<hdd device path>+ too?